src/RestWrite.js

```diff
@@ -46,18 +46,6 @@ function RestWrite(
   this.storage = {};
   this.runOptions = {};
   this.context = {};
-  if (!query && data.objectId) {
```

Comment:
I'd consider adding new parse-server option allowing to skip this check ( launch --flag + ENV variable) instead of just removing it for everyone.

Reply:
Maybe like `enableCustomId` or `enableCustomObjectId` ?

Reply:
yep something like that, or may be `allow` is more precise since it just removes the restriction: 
 `allowManualObjectIds` - i'd stick with this one
 `allowCustomObjectId`  
 `allowClientObjectIds`

Reply:
I agree with you guys! Can we join forces and show me the way to do this? I need some shortcuts to where it should be implemented. Maybe you can join me on the edits?

Reply:
@BufferUnderflower Great! it follows `allowClientClassCreation` and `allowHeaders` convention.